universities.properties

```diff
@@ -19,7 +19,7 @@
 天津大学 =	Tianjin University
 山东大学 =	Shandong University
 吉林大学 =	Jilin University
-南方科技大学 =中国	South China University of Technology
+南方科技大学 = South China University of Technology
 厦门大学 =	Amoy University
 北京师范大学 =	Beijing Normal University
 北京理工大学 =	Beijing Institute of Technology
```

Comment:
I guess you were trying to align the codes in this file, but it seems like they are not aligned completely :(